google/cloud/security/inventory/inventory_loader.py

```diff
@@ -32,62 +32,46 @@
       --email_recipient <email address of the email recipient> (optional) \\
       --loglevel <debug|info|warning|error>
 
+TODO: Move all these flags to the config file.
```

Comment:
nit: We'll need to put the todo before the pydoc because gflags uses the pydoc for --helpfull _I think_